jython/InitAllTurnoutsSensors.py

```diff
@@ -24,6 +24,11 @@
 #######    WE HAVE LEARNED THE HARD WAY, THAT PANEL FILES THAT ACTIATE Turnouts
 #######    AT LOAD TIME, RISK THE CONSEQUENCES OF MOVING A Turnout UNDER A TRAIN
 
+# This script kicks off an independent series of actions what will take some time to 
+# complete once the script itself completes.  If you are running this script as a 
+# startup action, you might need to include a pause startuo action before you run any
```

Comment:
No need to change, but startuo is misspelled. This applies to all the changed files.
`pause startuo action`